subprojects/packagefiles/cfitsio/meson.options

```diff
@@ -0,0 +1,38 @@
+option(
+    'curl',
+    type: 'feature',
+    description: 'Enable linking with the curl library. Enables remote file i/o support',
+)
+option(
+    'reentrant',
+    type: 'feature',
+    description: 'Enable reentrant multithreading',
+)
+option(
+    'utils',
+    type: 'boolean',
+    description: 'Build fpack, funpack, fitscopy, and fitsverify executables',
```

Comment:
... and as we see here the option does indeed claim it will control building, not control installing.